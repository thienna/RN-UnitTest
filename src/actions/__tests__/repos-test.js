import { selectRepo, SELECT_REPO } from '../repos';

it('creates a SELECT_REPO action', () => {
  // You do it
  expect(selectRepo(1000)).toEqual(
    {
      type: SELECT_REPO,
      id: 1000
    }
  );
  // Jest does it
  expect(selectRepo(1000)).toMatchSnapshot();
});