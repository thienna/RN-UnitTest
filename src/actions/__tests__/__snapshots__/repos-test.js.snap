// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a SELECT_REPO action 1`] = `
Object {
  "id": 1000,
  "type": "SELECT_REPO",
}
`;
